src/common/types/cpuArchitecture.ts

```diff
@@ -1,25 +1,71 @@
+import { SupportLevels } from '../api';
+
 export enum CpuArchitecture {
   x86 = 'x86_64',
   ARM = 'arm64',
   MULTI = 'multi',
   // This value refers to use whichever CPU architecture was selected for the cluster
   // It's not a value that can be returned from the Backend.
   USE_DAY1_ARCHITECTURE = 'cluster-day1-cpu-architecture',
+  ppc64le = 'ppc64le',
+  s390x = 's390x',
 }
 export enum OcmCpuArchitecture {
   x86 = 'amd64',
   ARM = 'arm64',
   MULTI = 'multi',
+  ppc64le = 'ppc64le',
+  s390x = 's390x',
 }
 
+type CpuArchitectureFeatureIds =
```

Comment:
I'd add a comment mentioning that these need to be equal to the FeatureId and transformed etc